core/domain/user_services_test.py

```diff
@@ -1899,6 +1934,30 @@ def test_set_user_has_viewed_lesson_info_modal_once(self):
         self.assertTrue(
             user_settings_model.has_viewed_lesson_info_modal_once)
 
+    def test_log_username_change(self):
+        def mock_get_current_time_in_millisec():
+            return 1.00
+
+        with self.swap(
+            utils, 'get_current_time_in_millisecs',
+            mock_get_current_time_in_millisec):
+            committer_id = 'someUser'
+            audit = audit_models.UsernameChangeAuditModel()
+            model_id = '%s.%d' % (
```

Comment:
Is this the expected model ID?

I think a better test would probably be: have a pre-check fetching all audit models and confirming there are none, and have a post-check fetching all audit models and confirming there is one, and then test the properties of that fetched model.

Reply:
I should have made changes that follow your suggestion. Please check!